src/redux/reducers.js

```diff
@@ -1,11 +1,11 @@
 import App from './App/reducer';
-import Dashboard from './Dashboard/reducer';
+import dashboard from './Dashboard/reducer';
```

Comment:
is this proper naming convention for lowercase?

Reply:
yep

Reply:
in majority of apps I've read and written

Reply:
👍